src/PanthereTestCaseTrait.php

```diff
@@ -52,7 +52,29 @@
      */
     protected static $panthereClient;
 
+    /**
+     * @var bool
+     */
+    private static $stopServerOnTeardown = true;
```

Comment:
Can't you use a public property instead? So we can get rid of the `stop` and `keep` methods?

Reply:
It's called by the `ServerListener`, and I don't know if I can access the instantiated test from the listener :confused:

Reply:
You can keep the property static and call `PantherTestCase::$stopServerOnTeardown = false` right?

Reply:
Do you mean to make the property public?